packages/noya-state/src/reducers/pointReducer.ts

```diff
@@ -14,7 +15,12 @@ import { ApplicationState, SetNumberMode } from './applicationReducer';
 export type PointAction =
   | [type: 'setPointCurveMode', curveMode: Sketch.CurveMode]
   | [type: 'setPointCornerRadius', amount: number, mode?: SetNumberMode]
-  | [type: 'setPointX' | 'setPointY', amount: number, mode?: SetNumberMode]
+  | [
+      type: 'setPointX' | 'setPointY',
+      amount: number,
+      mode?: SetNumberMode,
+      direction?: 'start' | 'end',
```

Comment:
Maybe this would be simpler if we pass in the `SelectedPointLists`? We can make it a required argument, and move it to position 2 in the tuple so the API looks more consistent with others that take ids.

```ts
[
      type: 'setPointX' | 'setPointY',
      pointLists: SelectedPointLists,
      amount: number,
      mode?: SetNumberMode,
]
```